src/components/index.tsx

```diff
@@ -1,3 +1,3 @@
-import Button from "./Button/Button";
+import Button from "../components/Button/Button";
```

Comment:
import Button from "./Button/Button";

This is correct, there is no need to go a level back ../components